src/2024h2/async_fn_everywhere.md

```diff
@@ -110,13 +110,12 @@ The goal for the year is to stabilize a solution to the ["send bound" problem][s
 
 #### Author draft RFC for async vision
 
-We plan to revise the [Async Vision Doc][AVD] and restructure it as a draft RFC, most likely to be approved by the [Lang][] and [Libs-API][] teams (we do not necessarily expect that RFC to be accepted by end of year). Our observation is that the previous version of the async vision doc, which was never RFC'd, never attained the legitimacy of being the "plan of record". In addition, a number of things have changed in the intervening years (for example, async functions in traits are now stable) and we are in a position to identify clearer next steps. The 
+We plan to revise the [Async Vision Doc][AVD] and restructure it as a draft RFC, most likely to be approved by the [Lang][] and [Libs-API][] teams (we do not necessarily expect that RFC to be accepted by end of year). Our observation is that the previous version of the async vision doc, which was never RFC'd, never attained the legitimacy of being the "plan of record". In addition, a number of things have changed in the intervening years (for example, async functions in traits are now stable) and we are in a position to identify clearer next steps.
 
 [AVD]: https://rust-lang.github.io/wg-async/vision.html
 
 This RFC will lay out a "plan of attack" for async, including both obvious good things (similar to [async closures][]) but also "known unknowns" and ways to resolve them. Areas the RFC is expected to cover are as follows:
```

Comment:
Where should `async closures` link to here?

The "Async closures" item in the "Development in the Async Rust ecosystem is stalled by missing language features and interop traits" section can't be used as an anchor, as it's not a `###` title.

Maybe the link can be removed?

```suggestion
This RFC will lay out a "plan of attack" for async, including both obvious good things (similar to async closures) but also "known unknowns" and ways to resolve them. Areas the RFC is expected to cover are as follows:
```

Reply:
The issue persists but this file has been renamed since I've opened the PR.